src/js/renderControls.js

```diff
@@ -81,8 +81,15 @@ function renderControls() {
               <span class="sr-only">Share via email</span>
             </a>
           </li>
+          <li>
+            <a id="copy-short-url" class="copy" onclick={"copyShortUrl()"} >
```

Comment:
While this will totally work, can you please use `click={copyShortUrl}` instead?